src/inspector_agent.cc

```diff
@@ -87,7 +87,6 @@ inline void* StartIoThreadMain(void* unused) {
     if (agent != nullptr)
       agent->RequestIoThreadStart();
   }
-  return nullptr;
```

Comment:
Should this function return `void` instead of `void*` now?

Reply:
A function pointer to `StartIoThreadMain` is passed to `pthread_create`, which expects the return type to be `void*`.